src/App.vue

```diff
@@ -104,25 +104,41 @@ export default class App extends Vue {
 
   sendNotifications() {
     if (((this.minutes === this.previousPeriod.end - this.$store.state.settings.endTime) && (!this.$store.state.settings.notificationSent) && (this.$store.state.settings.startorend !== 'start')) && (this.$store.state.settings.endTime === 0)) {
```

Comment:
Not a general comment on the `playSound` function, but I've noticed a lot of redundancy in these `if-else` case statements. For example, I feel like you can make a larger `if` statement that takes out `this.$store.state.settings.notificationSent`.

Reply:
Aight I will look into it. The purpose of this.$store.state.settings.notificationSent is so that a notification is not sent every 5 seconds within the minute that the period ends.

Reply:
In regards to `!this.$store.state.settings.notificationSent`, you could probably have it as the top if statement, then you could nest all the other if statements in it.